components/CtaLink/src/index.scss

```diff
@@ -16,7 +16,6 @@
   padding-inline-start: var(--denhaag-cta-link-padding-inline, calc(var(--denhaag-cta-link-gap) / 1.5));
   padding-inline-end: var(--denhaag-cta-link-padding-inline, calc(var(--denhaag-cta-link-gap) / 1.5));
   text-decoration: none;
-  word-break: break-word;
```

Comment:
<img width="292" alt="Screenshot 2022-08-16 at 15 03 51" src="https://user-images.githubusercontent.com/877246/184886384-ddf9c18a-7893-44b9-a576-626ed91d5446.png">

The `break-word` option was causing the words to be unreadable when the screen was small enough to actually break them.

@paulAcato I believe you've added this before to solve a problem, do you remember what it solved so we can ensure this will not reintroduce it?

Reply:
This can be removed. Can't find anymore why this was added.